int8 arm_index
bool relative
float32 vel
float32[] joints
---
int32 result
string strRt